asm/main.s

```diff
@@ -81,10 +81,10 @@ _02000918: .4byte 0x027E0000
 _0200091C: .4byte 0x00000400
 _02000920: .4byte 0x05000000
 _02000924: .4byte 0x07000000
-_02000928: .4byte 0x02000B68
+_02000928: .4byte _02000B68
 _0200092C: .4byte 0x027FFF9C
 _02000930: .4byte 0x01FF8000
-_02000934: .4byte 0x02000C55 @ NdsMain
+_02000934: .4byte NdsMain+1
```

Comment:
Can you explain the `+1` here? As far as I know, NdsMain starts at 0x02000c54, why is it offset?

Reply:
Thumb functions add 1 because of the thumb flag